packages/client/src/modules/app/app.tsx

```diff
@@ -54,6 +55,12 @@ export default function App() {
     setCountryInQueryParam(initialCountry);
   }
 
+  useEffect(() =>
+    process.env.NODE_ENV === 'production'
```

Comment:
Can we move this out of the component and add at the start of the index.tsx file may be as this is not related to a component but something that needs to be done at the entry point

Reply:
I have moved it to index.tsx. 
But I am interested to know, whether App component's successful load should be tracked OR index.tsx & app component are not very much different in terms of the application load cycle

Reply:
actually there is no need for us to track whether the app component or index component loaded as we are just interested in whether the page was loaded